travis-build/test.sh

```diff
@@ -64,7 +67,7 @@ for l in sys.stdin:
     m = p.search(l)
     if m:
         cases.add(m.group(1).split(':')[-1])
-print '\n'.join(cases)
+print('\n'.join(cases))
```

Comment:
print used as function, should better add:

``from __future__ import print_function``